### ISSUE TITLE
Translation API

### ISSUE THREAD

**Issue by boformer** [SpongePowered/SpongeDocs#192]
Documentation for the new Translation API is needed:

https://github.com/SpongePowered/SpongeAPI/pull/563

**Comment by 12AwsomeMan34** [comment]
I'd imagine that the above PR will become obsolete once https://github.com/SpongePowered/SpongeAPI/pull/1088 is merged.

@windy1 do you plan to tackle this, seeing as that is your PR (once it's merged, of course)?

**Comment by windy1** [comment]
Yes I will once it's merged however I'm not sure when that'll be since that PR has kind of taken a back seat lately.

**Comment by ST-DDT** [comment]
The latest variant is currently [this #1224](https://github.com/SpongePowered/SpongeAPI/pull/1224).
However its still not implemented.

**Comment by Inscrutable** [comment]
That PR has been closed, and the whole kaboodle was kicked into the too-hard basket of API 8.
The only folk who know the present status of this are @ryantheleach or @kashike
